src/cpp/src/image_generation/models/t5_encoder_model.cpp

```diff
@@ -93,10 +93,27 @@ ov::Tensor T5EncoderModel::infer(const std::string& pos_prompt, int max_sequence
         "infer's max_sequence_length ", max_sequence_length);
 
     if (input_ids_shape[0] == 0 || input_ids_shape[1] == 0) {
-        input_ids.set_shape({1, static_cast<size_t>(max_sequence_length)});
+        if (do_classifier_free_guidance) {
+            input_ids.set_shape({2, static_cast<size_t>(max_sequence_length)});
```

Comment:
let's introduce variable called `batch_size = do_classifier_free_guidance ? 2 : 1;` and re-use it here

Reply:
Done